### PR TITLE
feat: ibc fees NTRN-254

### PR DESCRIPTION
**Pull request by ratik**
https://p2pvalidator.atlassian.net/browse/NTRN-254

integration tests: https://github.com/neutron-org/neutron-integration-tests/pull/32

### CODE REVIEW

**Comment by foxpy** [comment]
Wait… Isn't it supposed to be NTRN-254?

File: contracts/neutron_interchain_txs/src/contract.rs [hunk 1]
```diff
@@ -187,6 +193,28 @@ fn msg_with_sudo_callback<C: Into<CosmosMsg<T>>, T>(
...
 
+fn execute_set_fees(
+    deps: DepsMut,
+    denom: String,
+    _recv_fee: u128,
+    ack_fee: u128,
+    timeout_fee: u128,
+) -> StdResult<Response<NeutronMsg>> {
+    let fees = IbcFee {
+        recv_fee: vec![],
```
Comment (foxpy) [inline contracts/neutron_interchain_txs/src/contract.rs:204]: Why don't you use `_recv_fee` here? :thinking:

**Review by foxpy — CHANGES_REQUESTED** [review]

File: contracts/neutron_interchain_txs/src/contract.rs (hunk 1 above)
Comment (ratik) [reply contracts/neutron_interchain_txs/src/contract.rs]: seems copy-paste issue

**Review by foxpy — CHANGES_REQUESTED** [review]
Please specify in PR comment that this PR should be merged with https://github.com/neutron-org/neutron/pull/83

**Review by foxpy — APPROVED** [review]

**Review by pr0n00gler — APPROVED** [review]

**Review by oopcode — APPROVED** [review]

**Review by quasisamurai — APPROVED** [review]

**Review by sotnikov-s — APPROVED** [review]

**Review by pr0n00gler — APPROVED** [review]

**Review by swelf19 — APPROVED** [review]

**Pull request merged**